runtime/autoload/man.vim

```diff
@@ -116,7 +116,11 @@ function! s:get_page(path) abort
   " Force MANPAGER=cat to ensure Vim is not recursively invoked (by man-db).
   " http://comments.gmane.org/gmane.editors.vim.devel/29085
   " Respect $MANWIDTH, or default to window width.
-  return s:system(['env', 'MANPAGER=cat', (empty($MANWIDTH) ? 'MANWIDTH='.winwidth(0) : ''), 'man', a:path])
```

Comment:
Update would be great. Actually, I was hoping you'd test it first to make sure it works for you.